broker/tests/unit/plugin/kubejobs/test_plugin.py

```diff
@@ -236,45 +324,40 @@ def test_wrong_request_body(self):
 
 class TestKubeJobsProvider(unittest.TestCase):
 
-    """
-    Set up KubeJobsProvider objects
-    """
-
     def setUp(self):
+        """
+        Set up KubeJobsProvider objects
+        """
         self.provider1 = KubeJobsProvider()
         self.provider2 = KubeJobsProvider()
 
     def tearDown(self):
         pass
 
-    """
-    Test the Get Title of the KubeJobs Provider
-    """
-
     def test_get_title(self):
+        """
+        Test the Get Title of the KubeJobs Provider
+        """
         self.assertEqual(self.provider1.get_title(),
                          'Kubernetes Batch Jobs Plugin')
         self.assertEqual(self.provider2.get_title(),
                          'Kubernetes Batch Jobs Plugin')
 
-    """
-    Test the Get Description of the KubeJobs Provider
-    """
-
     def test_get_description(self):
+        """
+        Test the Get Description of the KubeJobs Provider
+        """
         self.assertEqual(
             self.provider1.get_description(),
             'Plugin that allows utilization of Batch Jobs over a k8s cluster')
         self.assertEqual(
             self.provider1.get_description(),
```

Comment:
Here is written provider1 twice, shouldn't it be provider 2?

Reply:
Fixed